deploy/package.sh

```diff
@@ -65,7 +65,7 @@ if [[ -z "$SYNAPSE_POOL" ]]; then
 fi
 
 
-MODE="$([[ $DEPLOY_PGSQL = "false" ]] && echo "no-postgres")"
+MODE="batch-account$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres")"
```

Comment:
Change to:

MODE="batch-account$([[ $DEPLOY_PGSQL = "false" ]] && echo ",no-postgres" || echo '')"